tests/test_examples.py

```diff
@@ -81,7 +81,7 @@ def test_collection_examples(self, ont_gridion):
         # different result on your iRODS, depending on your user and zone.
         assert coll.permissions() == [AC("irods", Permission.OWN, zone="testZone")]
 
-    def test_data_object_examples(self, ont_gridion):
+    def test_data_object_examples(self, ont_gridion, tmp_path):
```

Comment:
How does `tmp_path` get mapped into the container?  Are you also mapping `TMPDIR` and pytest respects that?